server/channels/db/migrations/mysql/000126_sharedchannels_remotes_add_deleteat.up.sql

```diff
@@ -0,0 +1,44 @@
+SET @preparedStatement = (SELECT IF(
+    (
+        SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'RemoteClusters'
+        AND table_schema = DATABASE()
+        AND column_name = 'DeleteAt'
+    ) > 0,
+    'SELECT 1',
+    'ALTER TABLE RemoteClusters ADD DeleteAt bigint(20) DEFAULT 0;'
+));
+
+PREPARE alterIfNotExists FROM @preparedStatement;
+EXECUTE alterIfNotExists;
+DEALLOCATE PREPARE alterIfNotExists;
+
+SET @preparedStatement = (SELECT IF(
+    (
+        SELECT COUNT(*) FROM INFORMATION_SCHEMA.STATISTICS
+        WHERE table_name = 'RemoteClusters'
+        AND table_schema = DATABASE()
+        AND index_name = 'remote_clusters_site_url_unique'
+    ) > 0,
+    'DROP INDEX remote_clusters_site_url_unique ON RemoteClusters;',
```

Comment:
Minor nit:  in general I would think deleting the index before adding the column would perform better.  Not issue for the size of this table.

Reply:
Right, good catch! Fixed in [432eef9](https://github.com/mattermost/mattermost/pull/28159/commits/432eef9ee8bfeba735f06f09d7fa4718e1a1663e)